warehouse/query-core/src/main/java/datawave/query/discovery/DiscoveryLogic.java

```diff
@@ -369,6 +372,9 @@ public ShardIndexQueryTable clone() {
             String literal = literalAndField.getKey(), field = literalAndField.getValue();
             // if we're _ANYFIELD_, then use null when making the literal range
             field = Constants.ANY_FIELD.equals(field) ? null : field;
+            if (field != null) {
+                familiesToSeek.add(new Text(field));
+            }
```

Comment:
Yikes!  Good catch.  Does this mean we were potentially returning things that we shouldn't have before this fix?